sympy/functions/elementary/trigonometric.py

```diff
@@ -375,7 +375,7 @@ def _eval_rewrite_as_Pow(self, arg):
             return I*x**-I / 2 - I*x**I /2
 
     def _eval_rewrite_as_cos(self, arg):
-        return -cos(arg + S.Pi/2)
+        return -cos(arg + S.Pi/2, evaluate=False)
```

Comment:
Of course, there is no compelling reason for the change. But perhaps you also feel that it would look better. (There are similar constructs in other places.)